shared/schemas/workflow-ref-schema.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "WorkflowRef",
+    "description": "Workflow reference",
+    "type": "object",
+    "properties": {
+        "id": {
```

Comment:
Do we need a link here? I guess it can be determined from the ID.

Reply:
The ID should be enough. 
Perhaps in the future we'll have to support more than a single Sonataflow stack and then we'll need another identifier for it, but this schema doesn't prevent from extending it.